Add vuecal-cell-(viewId)-xxx class to cell

CarlosOnline opened the pull request:
Add class `vuecal-cell-(view.id)-(datePart)` to cell.

Goal is to allow styling of an entire column in month view.  For example Saturday will have a special background color.

```
<div class="vuecal__cell vuecal-cell-month-5">
    <div column="" tabindex="0" aria-label="4" class="vuecal__flex vuecal__cell-content">
           <div class="vuecal__cell-date">4</div><!----><!----><!----></div></div>
```

antoniandre commented:
Hi @CarlosOnline, 
Thanks for your contribution!

I like the idea but I will refactor this once pulled. 
Nothing serious, a little improvement and the CSS class should follow the BEM convention.
Will let you know when I release it!

antoniandre commented:
Now released in 3.2.1.

For info, I changed the days of the week to 1 to 7 instead of 0 to 6 to be consistent, and moved the code where it belongs.
The final class to use will be `.vuecal__cell--day[day]`. Only available on month view, as I don't think it makes much sense on other views.

Cheers!

CarlosOnline commented:
Thanks I appreciate it.

On Wed, Apr 22, 2020, 7:22 AM Antoni <notifications@github.com> wrote:

> Now released in 3.2.1.
>
> For info, I changed the days of the week to 1 to 7 instead of 0 to 6 to be
> consistent, and moved the code where it belongs.
> The final class to use will be .vuecal__cell--day[day]. Only available on
> month view, as I don't think it makes much sense on other views.
>
> Cheers!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/antoniandre/vue-cal/pull/272#issuecomment-617809980>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA4FK54KYPWOD3OV6PIMSDLRN34UDANCNFSM4ML4QZ5Q>
> .
>
